index.html

```diff
@@ -64,113 +78,138 @@ <h2 class="title-about">I'm a Software Developer</h2>
+            <h2>Web Portfolio</h2>
+            <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="card-lang">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+            </ul>
+            <a href="#" class="btn-works">See Project</a>
+        </div>
+        <div class="card card-4">
+            <h1 class="card-title">Profesional Art Printing Data</h1>
+            <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="card-lang">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+            </ul>
+            <a href="#" class="btn-works">See Project</a>
+        </div>
+        <div class="card card-5">
+            <h1 class="card-title">Profesional Art Printing Data</h1>
+            <h2 class="card-desk-title">Data Dashboard Healthcare</h2>
+            <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="card-lang">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>                
+            </ul>
+            <a href="#" class="btn-works">See Project</a>
+        </div>
+        <div class="card card-6">
+            <h1 class="card-title">Profesional Art Printing Data</h1>
+            <h2>Web Portfolio</h2>
+            <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+            <ul class="card-lang">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+            </ul>
+            <a href="#" class="btn-works">See Project</a>
+        </div>
+        
+
+    </section>
+
+    <!-- About Me Section -->
+
     <section class="about">
 
-        <section class="introduction">
-            <h1>About me</h1>
-        
-            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <div class="introduction">
+
+            <div class="present-ab">
+                <h1>About me</h1>
         
-            <a class="get-resume" href="#">Get my resume</a>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            
+                <a class="get-resume" href="#">Get my resume</a>
+            </div>
+            
```

Comment:
Kindly implement the following changes for your about-section :

- please change the background color to `#262626` as given in the design
- kindly use the right font size, margin, and padding for the text and the button as well so it matches the design
- kindly use the thickness of the horizontal line you used in your design
- lastly, please align the image to be in the same size and similar to the design

 Figma design :
![image](https://github.com/ulises2607/Portfolio/assets/20537712/50e7ee23-c90a-4310-8be7-db6658dc9eaa)

Your design :
![image](https://github.com/ulises2607/Portfolio/assets/20537712/fd290f99-f635-4198-bbd4-bfd25abf0e69)